modules/machineset-azure-capacity-reservation.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-azure.adoc
+
+ifeval::["{context}" == "cpmso-config-options-azure"]
+:cpmso:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineset-azure-capacity-reservation_{context}"]
+= Configuring Capacity Reservation by using machine sets
+
+//Note: This is in 4.16.3, and might be backported through 4.14.z at a later date.
+//Earlier versions of OCP should get an update .z version here. OCP 4.17 docs should use {product-version}.
+{product-title} version 4.16.3 and later supports on-demand Capacity Reservation with Capacity Reservation groups on {azure-full} clusters.
+
+You can configure a machine set to deploy machines on any available resources that match the parameters of a capacity request that you define.
+These parameters specify the VM size, region, and number of instances that you want to reserve.
+If your {azure-short} subscription quota can accommodate the capacity request, the deployment succeeds.
+
+For more information, including limitations and suggested use cases for this {azure-short} instance type, see the {azure-full} documentation about link:https://learn.microsoft.com/en-us/azure/virtual-machines/capacity-reservation-overview[On-demand Capacity Reservation].
+
+[NOTE]
+====
+You cannot change an existing Capacity Reservation configuration for a machine set.
```

Comment:
```suggestion
You cannot change a Capacity Reservation configuration for a machine set.
```
In many cases, you can remove the word "existing", and the content will retain its meaning. Feel free to ignore this suggestion if it affects accuracy.

Reply:
This one I think does affect accuracy, but yeah thanks for pointing these out, my tendency is to overuse them lol